### DISCUSSION TITLE
Dịch freeze/unfreeze (weights)

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#109] [Terminology] [answered]
`freeze` dịch là `đóng băng` ok rồi, còn `unfreeze` dịch là `bỏ đóng băng` hay `rã đông` nhỉ?

cc @mlbvn/handson-ml

**Comment by cuongvng** [comment]
`rã băng`?

**Comment by rootonchair** [comment, marked as answer]
Em vote `bỏ đóng băng` do đơn giản và có vẻ hợp khi sử dụng trong câu. `rã` trong `rã đông` được hiểu là `decay` thì không hợp lắm

"Ta đóng băng các trọng số rồi sau đó bỏ đóng băng để tiếp tục huấn luyện"
